tests/e2e/initialization/node/main.go

```diff
@@ -55,11 +58,16 @@ func main() {
 		panic("rpc-servers is required, separated by commas")
 	}
 
+	persistenrPeers := strings.Split(persistentPeersStr, ",")
+	if len(persistenrPeers) == 0 {
+		panic("persistent peers are required, separated by commas")
+	}
```

Comment:
Where are these set into the node configs rn?

Reply:
Also, it seems to me that there is some initialization logic that we don't need anymore. For example, the file we are discussing right now. This file is meant to be used for initializing a single node via Docker by running this as a Go program.

Contrary to the full chain, we never ended up needing initialziing single nodes via containers. Instead, this is done by directly [calling Go functions](https://github.com/osmosis-labs/osmosis/blob/90891d6cb9462e51655ed851ef0cb85d6d5503dd/tests/e2e/e2e_test.go#L152).

We can do this because state sync always starts after the upgrade tests.